### PR TITLE
IE11. Permission denied error for http://www.zillow.com/ (close #433)

### LINKED ISSUE #433: IE11. Permission denied error for http://www.zillow.com/
```javascript
TypeError: Permission denied   
at emit (http://localhost:1401/hammerhead.js:10081:109)   
at _raiseStorageChanged (http://localhost:1401/hammerhead.js:7793:16)   
at _checkStorageChanged (http://localhost:1401/hammerhead.js:7754:67)   
at Anonymous function (http://localhost:1401/hammerhead.js:7723:20)
```

### PR DESCRIPTION
**Pull request by churkin**
Unfortunately it is very difficult to write the test.

\cc @VasilyStrelyaev @LavrovArtem @miherlosev

### CODE REVIEW

File: src/client/sandbox/base.js [hunk 1]
```diff
@@ -14,6 +14,11 @@ export default class SandboxBase extends EventEmitter {
     // NOTE: The sandbox is deactivated when its window is removed from the DOM.
     isDeactivated () {
         try {
+            // NOTE: In the IE11 situation may occur when the document is not active.
```
Comment (VasilyStrelyaev) [inline src/client/sandbox/base.js]:
```
            // NOTE: In IE11, a situation when the document is not active may occur.
```

Comment (VasilyStrelyaev) [inline src/client/sandbox/base.js]: though "situation may occur when" **may be** correct but I'm not 100% sure

**Comment by VasilyStrelyaev** [comment]
\r-

**Comment by churkin** [comment]
FPR

**Comment by VasilyStrelyaev** [comment]
lgtm

**Comment by LavrovArtem** [comment]
lgtm

**Pull request merged**